features/facebookImport/src/views/overview/overview.jsx

```diff
@@ -101,9 +101,10 @@ const Overview = () => {
                     }}
                     proceedButton={{
                         text: i18n.t("overview:new.import.dialog.continue"),
-                        onClick: () => {
+                        onClick: async () => {
+                            console.log(JSON.stringify(files));
```

Comment:
-log

Reply:
Dang it, thanks :D

Reply:
Even the best will be logged some days :D